docbook/Developers_Guide/en-US/Plugins_Building.xml

```diff
@@ -90,23 +101,216 @@ class ExamplePlugin extends MantisPlugin {
         $this->page = '';           # Default plugin page
 
         $this->version = '1.0';     # Plugin version string
-        $this->requires = array(    # Plugin dependencies, array of basename => version pairs
-            'MantisCore' => '1.2.0, &lt;= 1.2.0',  #   Should always depend on an appropriate version of MantisBT
-            );
+        $this->requires = array(    # Plugin dependencies
+            'MantisCore' => '1.3, &lt; 3.0',  # Should always depend on an appropriate
+                                           # version of MantisBT
+        );
 
         $this->author = '';         # Author/team name
         $this->contact = '';        # Author/team e-mail address
         $this->url = '';            # Support webpage
     }
 }
-				</programlisting>
+			</programlisting>
 
-				<para>
-					This alone will allow the Example plugin to be installed with MantisBT, and
-					is the foundation of any plugin.  More of the plugin development process
-					will be continued in the next sections.
-				</para>
-			</section>
+			<para>
+				This alone will allow the Example plugin to be installed with MantisBT, and
+				is the foundation of any plugin.  More of the plugin development process
+				will be continued in the next sections.
+			</para>
+		</section>
+
+		<section id="dev.plugins.building.properties">
+			<title>Properties</title>
+			<para>
+				This section describes the properties that can be defined when
+				registering the plugin.
+			</para>
+
+			<variablelist>
+
+				<varlistentry>
+					<term>name</term>
+					<listitem>
+						<para>Your plugin's full name.
+							<emphasis>Required value.</emphasis>
+						</para>
+					</listitem>
+				</varlistentry>
+
+				<varlistentry>
+					<term>description</term>
+					<listitem>
+						<para>A full description of your plugin.</para>
+					</listitem>
+				</varlistentry>
+
+				<varlistentry>
+					<term>page</term>
+					<listitem>
+						<para>The name of a plugin page for further information
+							and administration of the plugin.
+							This is used to create a link to the specified page
+							on Mantis' manage plugin page.
+						</para>
+					</listitem>
+				</varlistentry>
+
+				<varlistentry>
+					<term>version</term>
+					<listitem>
+						<para>Your plugin's version string.
```

Comment:
I think we should recommend and advertise support to semver, since when plugins have dependencies on each other, we also want to use semver semantics to check that the required dependencies are available.